### PR TITLE
Disable 'freelook' in the orthogonal mode.

### LINKED ISSUE #23078: 'freelook' navigation doesn't work in the orthogonal mode
**Godot version:**

0dbe01483a902c49ecedf4fd36b74353424145a5

**OS/device including version:**

Linux

**Issue description:**

'freelook' navigation is broken in orthogonal mode.
It should either:
  + **be disabled**
  + temporally jump in the perspective mode (Blender)

**Steps to reproduce:**
Switch to orthogonal mode (numpad-5)  and enable the the 'freelook' navigation (shift+f)

**Minimal reproduction project:**

### LINKED ISSUE #23095: Right-click ('freelook') does nothing useful in the orthogonal mode
**Godot version:**

0dbe01483a902c49ecedf4fd36b74353424145a5

**OS/device including version:**

Linux

**Issue description:**

RMB in the orthogonal 3D-view of the editor just hides and warps the mouse pointer.
It can pan the view (= shift + MMB).

**Steps to reproduce:**
Switch the 3D view to orthogonal mode (numpad + 5) and right-click anywhere in the view.

### PR DESCRIPTION
**Pull request by capnm**
Fixes #23095
Thanks @aaronfranke for hints.

Fixes #23078 

What this PR basically does is to fix the Godot editor
in the orthogonal 3D mode. It prevents cases in which
the cursor is locked or behaving strangely.

### CODE REVIEW

File: editor/plugins/spatial_editor_plugin.cpp [hunk 1]
```diff
@@ -1911,37 +1919,38 @@ void SpatialEditorViewport::_nav_orbit(Ref<InputEventWithModifiers> p_event, con
...
-	// Freelook only works properly in perspective.
-	// It could technically work in ortho, but it's terrible for a user due to FOV being a fixed width.
-	if (!orthogonal) {
-		real_t degrees_per_pixel = EditorSettings::get_singleton()->get("editors/3d/navigation_feel/orbit_sensitivity");
-		real_t radians_per_pixel = Math::deg2rad(degrees_per_pixel);
-		bool invert_y_axis = EditorSettings::get_singleton()->get("editors/3d/navigation/invert_y-axis");
-
-		// Note: do NOT assume the camera has the "current" transform, because it is interpolated and may have "lag".
-		Transform prev_camera_transform = to_camera_transform(cursor);
+	if (orthogonal) {
```
Comment (Zylann) [inline editor/plugins/spatial_editor_plugin.cpp]: Perhaps this condition should be moved out, `_nav_look` should do what it says, and same for `_nav_pan` ?

Comment (capnm) [reply editor/plugins/spatial_editor_plugin.cpp]: s/exception/alternative

Comment (Zylann) [reply editor/plugins/spatial_editor_plugin.cpp]: Hm nevermind, I was just a bit confused about why we would still be calling `_nav_look` in a situation where `NAV_LOOK` should not have been active in the first place.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**